core/src/test/java/org/infinispan/jmx/CacheMgmtInterceptorMBeanTest.java

```diff
@@ -151,6 +155,27 @@ public void testRemoves() throws Exception {
       assertRemoveHits(3);
       assertRemoveMisses(1);
    }
+   
+   public void testGetAll() throws Exception {
+      assertMisses(0);
+      assertHits(0);
+      assert 0 == advanced.getStats().getHits();
```

Comment:
I know the other methods are using asserts, however we are trying to move over affected code to org.testng.AssertJUnit.* methods instead if you could.  Thanks.